js/ui/speed_dial_action/speed_dial_item.js

```diff
@@ -81,18 +107,24 @@ const SpeedDialItem = Overlay.inherit({
 
         eventsEngine.off(overlayContent, eventName);
         eventsEngine.on(overlayContent, eventName, (e) => {
-            this._clickAction({ event: e, element: this.$element() });
+            this._clickActionArgs = {
+                event: e,
+                actionElement: this.element(),
+                element: this._getActionComponent().$element()
+            };
+
+            this._clickAction(this._clickActionArgs);
```

Comment:
Why you are defined `_clickActionArgs ` if it's used only once?

Reply:
Fixed